test/deno.test.ts

```diff
@@ -47,3 +53,30 @@ test("run deno test", async () => {
     );
   });
 });
+
+test("deno check", async () => {
+  jest.setTimeout(60_000);
+
+  if (!fs.existsSync("test/deno")) {
+    console.warn("skipping deno tests; run `yarn `compileForDeno`");
+    return;
+  }
+
+  return await new Promise<void>((resolve, reject) => {
+    execFile(
+      "deno",
+      ["eval", 'import * as edgedb from "./edgedb-deno/mod.ts"'],
+      {
+        env: process.env,
+      },
+      (error, stdout, stderr) => {
+        if (error) {
+          console.error(stderr);
+          reject(error);
+        }
+        console.log(stdout);
```

Comment:
Don't really need this `console.log`, nothing should get logged

Reply:
fixed